.github/workflows/tox.yml

```diff
@@ -27,11 +27,9 @@ jobs:
           - 9000:9000
 
     steps:
-    - uses: actions/checkout@v3
-      with:
```

Comment:
Doing this shallow clone breaks the automatic version computation done by `setuptools_scm`. This isn't causing tests to actually fail, but we should test the package in the same way that it will be released.

Reply:
Oops, I got this backwards. It was `release.yml` that was missing `fetch-depth: 0`.

Fixed in #155.